extras/clamav_log_report.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+#
+# Gather ClamAV scan log data from a list of AWS instances via SSM.
+#
+# Usage: ./clamav_log_report.sh <instance-id>...
+
```

Comment:
Is there any reason not to include our usual foo here?
```suggestion

set -o nounset
set -o errexit
set -o pipefail
```

Reply:
Nope- good catch, I just forgot about them, thanks!